Dockerfile

```diff
@@ -0,0 +1,17 @@
+# build stage
+FROM node:lts-alpine as build-stage
+WORKDIR '/app'
+RUN apk --no-cache add git
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+# production stage
+FROM nginx:stable-alpine as production-stage
+COPY --from=build-stage /app/dist /usr/share/nginx/html
+COPY nginx.conf /temp/nginx.conf 
+RUN envsubst / < /temp/nginx.conf > /etc/nginx/conf.d/default.conf
```

Comment:
I don't know if this line is necessary, ran into some problems and copy-pasted half a line as a solution